bundler.d/profiling.rb

```diff
@@ -0,0 +1,9 @@
+group :development do
```

Comment:
```suggestion
group :profiling, optional: true do
```

Reply:
The problem with introducing a new group is that it will never be available in development mode, because it is not automatically required in `config/application.rb`.
While in 'production' (if `Gemfile.in` present) it is required in code, when present.

Reply:
Huh, it catches `LoadError` so I'd assume it just continues if the group isn't present. We do this in production all the time. The `assets` group is practically never present, only when building.